docs/_basic/authenticating_oauth.md

```diff
@@ -10,7 +10,7 @@ Slack apps installed on multiple workspaces will need to implement OAuth, then s
 
 Bolt for JavaScript will create a **Redirect URL** `slack/oauth_redirect`, which Slack uses to redirect users after they complete your app's installation flow. You will need to add this **Redirect URL** in your app configuration settings under **OAuth and Permissions**. This path can be configured in the `installerOptions` argument described below.
 
-Bolt for JavaScript will also create a `slack/install` route, where you can find an `Add to Slack` button for your app to perform direct installs of your app. If you need any additional authorizations (user tokens) from users inside a team when your app is already installed or a reason to dynamically generate an install URL, you can generate it via `app.installer.generateInstallUrl()`. Read more about `generateInstallUrl()` in the [OAuth docs](https://slack.dev/node-slack-sdk/oauth#generating-an-installation-url). 
+Bolt for JavaScript will also create a `slack/install` route, where you can find an `Add to Slack` button for your app to perform direct installs of your app. If you need any additional authorizations (user tokens) from users inside a team when your app is already installed or a reason to dynamically generate an install URL, you can generate it by [defining a custom ExpressReceiver](https://slack.dev/bolt-js/concepts#receiver) and calling `receiver.installer.generateInstallUrl()`. Read more about `generateInstallUrl()` in the [OAuth docs](https://slack.dev/node-slack-sdk/oauth#generating-an-installation-url).
```

Comment:
As you don't need to implement your own custom `Receiver` for utilizing `generateInstallUrl` method, I don't think we should have the link to https://slack.dev/bolt-js/concepts#receiver that explains how to implement your own custom `Receiver`.

So, I think we can say "manually instantiating an `ExpressReceiver`, assigning the instance to a variable `receiver`, and then calling `receiver.installer.generateInstallUrl()`" without any links here.

Reply:
@seratch my mistake I meant to link to https://slack.dev/bolt-js/concepts#custom-routes instead but even that isn't perfect for this sentence. I like you solution of having no links.